iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractIdentifiable.java

```diff
@@ -42,7 +42,12 @@ public String getId() {
     }
 
     @Override
-    public String getName() {
```

Comment:
I think we should make getName deprecated to avoid a breaking change.
Then in a later release we could remove this getName, make getOptionalName deprecated and recreate a getName that behaves like getOptionalName. What do you think?

Reply:
Agree with you. I deprecated it and made it call `getNameOrId()` as default so we don't need to implement it in implementation classes.